akka-actor/src/main/scala/akka/io/UdpListener.scala

```diff
@@ -77,7 +77,9 @@ private[io] class UdpListener(
       log.debug("Unbinding endpoint [{}]", bind.localAddress)
       try {
         channel.close()
-        if (Helpers.isWindows) registration.enableInterest(OP_READ)
+        registration.enableInterest(OP_READ)
```

Comment:
is this intended? Should it just be set in the if statement below?

Reply:
Ups. Should only be below, duplicated now

Reply:
Should I test on Windows JDK11 with this removed?

Reply:
I don't think that matters, it will just enable the same interest twice.